birdspotter/birdspotter/dataio/scripts/get_user_datasets.py

```diff
@@ -45,32 +54,24 @@ def get_dataset_data(is_authed, uuid):
                           "comments"    : [i.comments for i in shapefile_lines],
                           }
     else:
-
-        precision = 3       # aggregate data to 3 decimals points of lat/long
-        precision_mod = 1  # allows for more precise tuning, >1 reduces region size <1 increases region size
-        aggregation = {}
-
+        aggregation = [0, 0, 0, ""]
+        
         for i in shapefile_lines:
             # separate by island name and by precision,
             # in case there are multiple islands in one dataset
-            key = (round(i.latitude*precision_mod, precision),
-                round(i.longitude*precision_mod, precision),
-                i.island_name)
-            if key in aggregation:
-                aggregation[key][0] += i.latitude
-                aggregation[key][1] += i.longitude
-                aggregation[key][2] += 1
-            else:
-                aggregation[key] = [i.latitude, i.longitude, 1, i.island_name]
+            
+            aggregation[0] += i.latitude
+            aggregation[1] += i.longitude
```

Comment:
This isn't quite the best way to do it but no need to hold back the branch for it